squirrel_safety/launch/squirrel_safety.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+  <node name="squirrel_safety_node" pkg="squirrel_safety" type="squirrel_safety_node.py" output="screen" respawn="true">
+    <param name="airskin_topic" value="/arm_bumper" type="string"/>
```

Comment:
I think the topic name is `/airskin/arm_bumper` https://github.com/squirrel-project/squirrel_driver/blob/8c0ffec4fda82687bf5237fcb1625ce665c5ded7/airskin/src/airskin_node.cc#L278